src/extension/devtools/devtools.ts

```diff
@@ -144,10 +142,9 @@ async function createDevtoolsPanel() {
         devtoolsMachine.send("retry");
       });
 
-      const subscription = devtoolsMachine.subscribe(({ value }) => {
+      devtoolsMachine.subscribe(({ value }) => {
```

Comment:
This fixes the bug mentioned in https://github.com/apollographql/apollo-client-devtools/pull/1283#discussion_r1532946252